packages/charting/src/types/IDataPoint.ts

```diff
@@ -267,9 +267,10 @@ export interface IVerticalStackedChartProps {
   xAxisPoint: number | string;
 
   /**
-   * chart title for the chart
+   * Callout data for x axis
+   * This is an optional prop, If haven't given, legend will take
    */
-  chartTitle?: string;
+  xAxisCalloutData?: string;
```

Comment:
Was this intentional? It seems like `xAxisCalloutData` should have been a *new* prop, not a replacement for `chartTitle`.

Reply:
`chartTitle` was not being used. It's not technically a replacement.